x-pack/plugins/integration_assistant/public/common/lib/api.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { HttpSetup } from '@kbn/core-http-browser';
+import type {
+  EcsMappingApiRequest,
+  EcsMappingApiResponse,
+  CategorizationApiRequest,
+  CategorizationApiResponse,
+  RelatedApiRequest,
+  RelatedApiResponse,
+  CheckPipelineApiRequest,
+  CheckPipelineApiResponse,
+  BuildIntegrationApiRequest,
+  InstallPackageResponse,
+  GetPackagesResponse,
+} from '../../../common/types';
```

Comment:
Can we not import them from `../../../common`

Reply:
it's the same, we only import types